LayoutKit/Internal/TextExtensions.swift

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import UIKit
+
+extension Text {
+
+    /// Calculate the text size within `maxSize` by given `UIFont`
+    func textSize(maxSize: CGSize,
```

Comment:
It is using `UIKit` and only for iOS and tvOS. `Text` is shared with `macOS`.

I will update the label.